src/transformers/tokenization_xlm_roberta.py

```diff
@@ -261,21 +269,40 @@ def create_token_type_ids_from_sequences(
 
     @property
     def vocab_size(self):
-        return len(self.sp_model) + len(self.fairseq_tokens_to_ids)
+        return len(self.sp_model) + self.fairseq_offset + 1  # Add the <mask> token
 
     def get_vocab(self):
         vocab = {self.convert_ids_to_tokens(i): i for i in range(self.vocab_size)}
         vocab.update(self.added_tokens_encoder)
         return vocab
 
+    def preprocess_text(self, inputs):
+        if self.remove_space:
+            outputs = " ".join(inputs.strip().split())
+        else:
+            outputs = inputs
+        outputs = outputs.replace("``", '"').replace("''", '"')
+
+        if not self.keep_accents:
+            outputs = unicodedata.normalize("NFKD", outputs)
+            outputs = "".join([c for c in outputs if not unicodedata.combining(c)])
+        if self.do_lower_case:
+            outputs = outputs.lower()
+
+        return outputs
```

Comment:
Does this method come from fairseq XLM-R code?
The reason why I ask is that it seems a bit English specific to me while XLM-R is supposed to be multilingual.

Reply:
You're correct, this may be too english specific. I was wondering if we even need to have an option to lowercase the text?
If we don't, let's remove that text preprocessing altogether.